test/Microsoft.TestPlatform.AcceptanceTests/TranslationLayerTests/RunTests.cs

```diff
@@ -196,9 +196,16 @@ public void RunTestsShouldThrowOnStackOverflowException(RunnerInfo runnerInfo)
             new TestPlatformOptions() { TestCaseFilter = "ExitWithStackoverFlow" },
             _runEventHandler);
 
-        var errorMessage = runnerInfo.TargetFramework == "net451"
-            ? $"The active test run was aborted. Reason: Test host process crashed : Process is terminated due to StackOverflowException.{Environment.NewLine}"
-            : $"The active test run was aborted. Reason: Test host process crashed : Process is terminating due to StackOverflowException.{Environment.NewLine}";
+        var errorMessage = "The active test run was aborted. Reason: Test host process crashed : ";
```

Comment:
Seems more complicated than what was done before. Any reason to just not invert the other condition?

Reply:
There was something wrong in the lines that got fixed later so I think that's just some left over from debugging/refactoring. I am reverting.